test.js

```diff
@@ -569,3 +569,16 @@ if (Promise.prototype.finally) {
 		t.is(result.message, 'called');
 	});
 }
+
+test('the spawned process is cancelable', t => {
+	const spawned = execa('ls');
+	spawned.catch(error => {
+		if (error.message === 'reason' && error.isCanceled) {
+			return;
+		}
+
+		throw error;
+	}); // To handle the CancelError
+	spawned.cancel('reason');
+	t.true(spawned.killed);
```

Comment:
I think `canceled` should be tested here instead of `killed`. See my other comment about adding that property.